.github/workflows/ci.yaml

```diff
@@ -0,0 +1,36 @@
+name: Python Package
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install build black
+          python -m pip install .[test]
+      - name: Lint
+        run: |
+          black --check -t py39 .
```

Comment:
You could use official black integration to see if the code matches the black code style.

https://black.readthedocs.io/en/stable/integrations/github_actions.html